shenyu-plugin/shenyu-plugin-mock/src/main/java/org/apache/shenyu/plugin/mock/generator/CnameGenerator.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shenyu.plugin.mock.generator;
+
+import java.util.Random;
+
+/**
+ * Chinese name generator.
+ */
+public class CnameGenerator extends AbstractGenerator<String> {
+    
+    private static final String[] FIRST_NAME = {"李", "王", "张", "刘", "陈", "杨", "赵", "黄", "周", "吴",
```

Comment:
whats mean？ it be remove

Reply:
Initialize some Chinese surnames and common characters in Chinese names, so that the randomly generated name looks more like a normal name, I will add some comments here